_dprhtml/js/dprviewmodel.js

```diff
@@ -65,48 +63,48 @@ class DprViewModel {
   }
 }
 
-const DPR_CMD_GOTO_PREV = 'gotoPrevCmd';
-const DPR_CMD_GOTO_INDEX = 'gotoIndexCmd';
-const DPR_CMD_GOTO_NEXT = 'gotoNextCmd';
-const DPR_CMD_GOTO_MYANMAR = 'gotoMyanmarCmd';
-const DPR_CMD_GOTO_THAI = 'gotoThaiCmd';
-const DPR_CMD_GOTO_RELM = 'gotoRelmCmd';
-const DPR_CMD_GOTO_RELA = 'gotoRelaCmd';
-const DPR_CMD_GOTO_RELT = 'gotoReltCmd';
-const DPR_CMD_COPY_PERMALINK = 'copyPermalinkCmd';
-const DPR_CMD_SEND_TO_CONVERTER = 'sendToConverter';
-const DPR_CMD_SEND_TO_TEXTPAD = 'sendToTextPad';
-const DPR_CMD_APPEND_TO_TEXTPAD = 'appendToTextpad';
-const DPR_CMD_SAVE_TO_DESKTOP = 'saveToDesktop';
-const DPR_CMD_SEARCH_IN_BOOK = 'searchInBook';
-const DPR_CMD_COPY_PLACE_TO_SIDEBAR = 'copyPlaceToSidebar';
-const DPR_CMD_BOOKMARK_SECTION = 'bookmarkSection';
-const DPR_CMD_TRANSLATE_ = 'translate';
-const DPR_CMD_TRANSLATE_0 = 'translate0';
-const DPR_CMD_TRANSLATE_1 = 'translate1';
-const DPR_CMD_TRANSLATE_2 = 'translate2';
-const DPR_CMD_TRANSLATE_3 = 'translate3';
-const DPR_CMD_TRANSLATE_4 = 'translate4';
-const DPR_CMD_TRANSLATE_5 = 'translate5';
-const DPR_CMD_TRANSLATE_6 = 'translate6';
-const DPR_CMD_TRANSLATE_7 = 'translate7';
-const DPR_CMD_TRANSLATE_8 = 'translate8';
-const DPR_CMD_TRANSLATE_9 = 'translate9';
-const DPR_CMD_TRANSLATE_10 = 'translate10';
-const DPR_CMD_ENTER_QUICK_REFERENCE = 'enterQuickReference';
-const DPR_CMD_OPEN_SETTINGS = 'openSettings';
-const DPR_CMD_GOTO_HOME = 'gotoHome';
-const DPR_CMD_GOTO_PREV_DICT_ENTRY = 'gotoPrevDictEntry';
-const DPR_CMD_GOTO_NEXT_DICT_ENTRY = 'gotoNextDictEntry';
-const DPR_CMD_TOGGLE_DPR_SIDEBAR = 'toggleDPRSidebar';
-const DPR_CMD_SHOW_BOTTOM_PANE = 'showBottomPane';
-const DPR_CMD_SHOW_PALI_QUOTE = 'showPaliQuote';
-const DPR_CMD_RESET_SETTINGS = 'resetSettings';
-const DPR_CMD_OPEN_NEW_QUIZZ = 'openNewQuizz';
-const DPR_CMD_OPEN_HELP = 'openHelp';
-const DPR_CMD_OPEN_HELP_VIDEO = 'openHelpVideo';
-const DPR_CMD_LAUNCH_FEEDBACK_FORM = 'launchFeedbackForm';
-const DPR_CMD_INSTALL_OFFLINE_APP = 'installOfflineApp';
+window.DPR_CMD_GOTO_PREV = 'gotoPrevCmd';

```

Comment:
I think I'd prefer having the commands namespaced under the window object - think it would help with organization. I'm tempted to suggest a namespace for all the DPR globals - but that might be excessive (sacrificing succintness)

Reply:
super - that is what I added the DPRGlobal class for. i also want to make most of the stuff write once considering it is a global variable.

i'll make that change before merging this commit

Reply:
Agree, since they're all defined in a single place, a global namespace seems redundant.

If you want to preserve the look and the succinctness, you can just convert the "DPD_CMD" prefix of the identifier into the namespace.

Reply:
i'll be doing this in the next commit as the size of this PR has increased quiet a bit due to infra stuff. let me know if that is no ok with you.

Reply:
Totally fine, no reason to have this PR too bloated.